Jenkinsfile

```diff
@@ -311,6 +311,25 @@ pipeline {
         }
     }
 
+    stage('L2: Computer Vision') {
+      when {
+        anyOf{
+          branch 'candidate'
+          changeRequest target: 'candidate'
+        }
+      }
+      failFast true
+      parallel {
+        stage ('MNIST image classification with LeNet-5 Test - on CPU') {
+          steps {
+            sh 'cd examples/cv && python mnist_lenet5_image_classification_pure_lightning.py trainer.gpus=0 \
+            trainer.fast_dev_run=true model.dataset.data_folder=/home/TestData'
+            sh 'rm -rf examples/cv/outputs'
+          }
```

Comment:
This should change exp_manager.root_dir to some custom path - say examples/CV/lenet_mnist and the rm below should remove that specific file. This is for future proofing against parallel ci runs which all target and read/write to same default root for.

Reply:
a) I have copied this from the other jenkins examples. b) This "test" runs on CPU.

Reply:
1) Look at ASR Jenkins block 2) Why is it even on cpu anyway ?

Reply:
still, removed 'rm'

Reply:
All CI machines have GPUs. I'm curious too - why not run this on GPU? Or GPU and CPU if you wish to test both. But only CPU seems strange.

Reply:
Removing RM will cause weights to persist in the CI env, and eventually it will run out of space. As I said, you should set the root_dir environment of exp_manager to a custom path, and rm -rf just that dir.

Reply:
This is simple: I am saving like 1s by not moving model to GPU takes time.

Why should I bother with GPU when I have a simple model and I just want to make sure that everything is properly connected and nothing is broken by e.g. changes in the classes up in the hierarchy.

Reply:
added `exp_manager.root dir`, changed to L0 (as it is really integration test)

Reply:
The script is not using exp_manager and I see no point in adding it - just for test.
So reverting to `rm -rf outputs` - as far as I understand it solves the problem of removing checkpoints.
Will solve the "parallel ci runs" when I will have more tests down there.